### PR TITLE
[release-v.12] Add missing installation watch for auth controller

### PR DESCRIPTION
**Pull request by rene-dekker**
V1.12 Add missing installation watch for auth controller
cherry pick of 786adf8ea5b4569ff3f803e4bd24908512b14b4a

### CODE REVIEW

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
